dapps/valuetransfers/packages/tangle/tangle.go

```diff
@@ -347,7 +535,7 @@ func (tangle *Tangle) solidifyPayload(cachedPayload *payload.CachedPayload, cach
 		}
 
 		// book the solid entities
-		transactionBooked, payloadBooked, bookingErr := tangle.book(currentCachedPayload.Retain(), currentCachedMetadata.Retain(), currentCachedTransaction.Retain(), currentCachedTransactionMetadata.Retain())
+		transactionBooked, payloadBooked, decisionPending, bookingErr := tangle.book(currentCachedPayload.Retain(), currentCachedMetadata.Retain(), currentCachedTransaction.Retain(), currentCachedTransactionMetadata.Retain())
```

Comment:
`solidifyPayload` is a freaking huge function. So, I lost track a bit tbh...
Can we somehow split this up and with this maybe also make the releases deferable to not have this repetition